exchanges.yml

```diff
@@ -217,17 +217,62 @@ components:
                     type: string
                   template:
                       type: string
-            workflowId:
+            id:
               type: string
-              description: the ID that will be used for the created workflow. workflowId is OPTIONAL.
+              description: The ID that will be used for the created workflow. Passing an ID is OPTIONAL.
     CreateWorkflowResponse:
       type: object
       additionalProperties: false
-      description: Object containing information about a created workflow.
+      description: Response containing the workflowData Object created.
       properties:
-        workflowId:
-          type: string
-          description: The URL that uniquely identifies the created workflow.
+        workflowData:
+          type: object
+          additionalProperties: false
+          description: Object containing information about a created workflow.
+          properties:
+            id:
+              type: string
+              description: The URL that uniquely identifies the created workflow.
+            stepInformation:
+              type: object
+              description: Information about the steps required for the workflow. Returning stepInformation is REQUIRED.
+              properties:
+                exchanges:
```

Comment:
I think this could be a potentially huge number of exchanges for a very popular workflow -- and it isn't part of the config, so we shouldn't do this as a requirement, though some implementation might return this kind of data if that is desirable/not problematic.